### PR TITLE
feat: Upgrade kratos chart to releaser 0.9.0

### PR DESCRIPTION
**Pull request by aeneasr**
## Checklist

- [ ] I have read the [contributing guidelines](../blob/master/CONTRIBUTING.md)
      and signed the CLA.
- [ ] I have referenced an issue containing the design document if my change
      introduces a new feature.
- [ ] I have read the [security policy](../security/policy).
- [ ] I confirm that this pull request does not address a security
      vulnerability. If this pull request addresses a security vulnerability, I
      confirm that I got green light (please contact
      [security@ory.sh](mailto:security@ory.sh)) from the maintainers to push
      the changes.
- [ ] I have added tests that prove my fix is effective or that my feature
      works.
- [ ] I have added necessary documentation within the code base (if
      appropriate).

### CODE REVIEW

File: helm/charts/kratos/README.md [hunk 1]
```diff
@@ -19,10 +19,10 @@ A ORY Kratos Helm chart for Kubernetes
 | deployment.extraVolumeMounts | list | `[]` |  |
 | deployment.extraVolumes | list | `[]` | If you want to mount external volume For example, mount a secret containing Certificate root CA to verify database TLS connection. |
 | deployment.labels | object | `{}` |  |
-| deployment.livenessProbe | object | `{"failureThreshold":5,"httpGet":{"path":"/health/alive","port":"http-admin"},"initialDelaySeconds":30,"periodSeconds":10}` | Configure the probes for when the deployment is considered ready and ongoing health check |
+| deployment.livenessProbe | object | `{"failureThreshold":5,"httpGet":{"path":"/admin/health/alive","port":"http-admin"},"initialDelaySeconds":30,"periodSeconds":10}` | Configure the probes for when the deployment is considered ready and ongoing health check |
```
Comment (Demonsthere) [inline helm/charts/kratos/README.md:22]: This file is autogenerated, no need to modify it in the PR ;)

**Review by Demonsthere — APPROVED** [review]

File: helm/charts/kratos/README.md (hunk 1 above)
Comment (aeneasr) [reply helm/charts/kratos/README.md:22]: Ok, I won't next time :)

**Comment by aeneasr** [comment]
Regarding release, we need to figure out how to ship this. We need this change in cloud relatively soon (today/tomorrow), but the version this targets is not yet released in the open source and its also incompatible with the older version of Ory Kratos.

**Comment by Demonsthere** [comment]
If this is a breaking change, that is connected with the 0.9 release, we can add a function to check the version of the image from tag, and do if 0.9 then x, else fallback to 0.8

**Comment by aeneasr** [comment]
Oh, that would be nice! Would this work for cloud also? If so, could you support me with that? :)

Alternatively I think we can also keep the old chart, as there is a 302 redirect from the old `/health/alive` to `/admin/health/alive` endpoints. So if 302 is ok for the health checker, it shouldn't be an issue I think.

**Comment by Demonsthere** [comment]
According to https://github.com/kubernetes/kubernetes/issues/73172 anything between 200 and 400 is accepted :)

**Comment by aeneasr** [comment]
Ok, in that case we can wait to merge this until 0.9 is out!

**Comment by aeneasr** [comment]
@Demonsthere 0.9 is out

**Pull request merged**